proxy/src/http/sql_over_http.rs

```diff
@@ -158,6 +158,19 @@ pub async fn handle(
         ("application_name", APP_NAME),
     ]);
 
+    // Determine the output options. Default behaviour is 'false'. Anything that is not
+    // strictly 'true' assumed to be false.
+    let raw_output = headers
+        .get("Neon-Raw-Text-Output")
+        .unwrap_or(&hyper::http::HeaderValue::from_str("false")?)
+        .to_str()?
```

Comment:
Maybe force the lowercase here?

Reply:
Such capitalization is default in http: https://en.wikipedia.org/wiki/List_of_HTTP_header_fields

Reply:
I don't think you need the unwrap_or, just should work (`HeaderValue` implements `PartialEq<str>`):

```rust
let raw_output = headers.get("Neon-Raw-Text-Output") == Some("true");
```

Re: casing, it might make sense to define a constants to get along to be faster with whatever normalization `http` does for headers (upper or lowercased):

```
static RAW_TEXT_OUTPUT = HeaderName::from_static("Neon-Raw-Text-Output");

static ARRAY_MODE: HeaderName = HeaderName::from_static("Neon-Array-Mode");
```

Then use these statics to query the header map. It is a bit difficult to follow what is the difference from code, but this seems heavily optimized. Internal representation is lowercased.

Reply:
Thank you, this way it is much more cleaner. Fixed.